src/dependencies.ts

```diff
@@ -150,7 +150,10 @@ export async function synchronizeBumps(
 
           let operatorTouse = operator;
 
-          if (saveExact) {
+          const useExactVersion =
+            releaseAs === ReleaseAsPresets.ALPHA || releaseAs === ReleaseAsPresets.BETA || preid || saveExact;
+
+          if (useExactVersion) {
```

Comment:
Interesting, I tried passing `saveExact: true` originally and it didn't seem to work. I notice there is not a direct test change in this PR to address it. I can manually test the build you made, but are we sure this will work?

Reply:
I've added tests in e8613ebb32ffac5de61433510312cb9349ddfbd2 to improve confidence. Let me know your thoughts